src/NServiceBus.RabbitMQ/Routing/ConventionalRoutingTopology.cs

```diff
@@ -79,6 +79,11 @@ public void DeclareAndInitialize(IModel channel, IEnumerable<string> receivingAd
                 channel.QueueDeclare(address, useDurableExchanges, false, false, null);
                 Initialize(channel, address);
             }
+
+            foreach (var address in receivingAddresses)
+            {
+                channel.ExchangeBind(address, "nsb.delay-destination", $"#.{address}");
```

Comment:
Sorry, failed to communicate. Haven't focused on tests yet.
Note to self: need to communicate better where we are and what is expected